api/src/data_migration/load/load_oracle_data_task.py

```diff
@@ -154,19 +154,19 @@ def do_insert(self, foreign_table: sqlalchemy.Table, staging_table: sqlalchemy.T
         t1 = time.monotonic()
         total_insert_count = sum(insert_chunk_count)
         self.increment("count.insert.total", total_insert_count)
-        self.increment(f"count.insert.{staging_table.name}", total_insert_count)
-        self.set_metrics(
-            {
-                f"count.insert.chunk.{staging_table.name}": ",".join(map(str, insert_chunk_count)),
-                f"time.insert.{staging_table.name}": round(t1 - t0, 3),
-            }
-        )
+
+        log_extra |= {
+            f"count.insert.{staging_table.name}": total_insert_count,
+            f"count.insert.chunk.{staging_table.name}": ",".join(map(str, insert_chunk_count)),
+            f"time.insert.{staging_table.name}": round(t1 - t0, 3),
+        }
```

Comment:
Have you ever used true metrics? You're using logs here, and describing them as "metrics" in the PR title. But I'm pretty sure what you actually want are true metrics:

https://docs.newrelic.com/docs/apm/agents/python-agent/supported-features/python-custom-metrics/

Reply:
I have used metrics before - the metrics we collect here follow a pattern from a past project where we did actually do that - it might be something we want to consider, but that requires some separate setup to do. Additionally, I found just having everything in "Logs" makes things easier rather than having to understand that your data lives in multiple places in New Relic.